### DISCUSSION TITLE
How to update cloud backup?

### DISCUSSION THREAD

**Discussion by edwyn-amador** [AutomaApp/automa#730] [Q&A] [answered]
I need to update cloud backup on some workflows, I don't see any option to update changes but only removing all the cloud backups of those first and do the process again, is this cloud backups update/sync, something possible already? If not could you please add it? Thanks!

**Comment by Kholid060** [comment, marked as answer]
If you have backup the workflows to the cloud, Automa will automatically sync the workflow to the cloud in the background when you edit the workflow in the extension.

**Reply by edwyn-amador** [reply]
Ohh, that's great, Thanks!
